class InvalidError(Exception):
	pass


class TodoNotFound(Exception):
	pass